### DISCUSSION TITLE
Question about updates.

### DISCUSSION THREAD

**Discussion by sannelo** [fnuecke/oc2#136] [General]
I apologize in advance for the question, due to the situation in the world. 
But when is the update planned?

**Comment by atirutw** [comment]
If you meant a new update to OC2: it's literally 12 days ago. You even can build the mod yourself.
If you meant this mod being released as stable: probably not any time soon

**Reply by sannelo** [reply]
I meant when is the new update for Open Computers 2 planned?

**Comment by fnuecke** [comment]
Not sure what you're asking for exactly here. If you're asking when this goes out of alpha: I don't know. If you're asking when the next minor version is released: I don't know. If you're asking if this is still being worked on: yes.

**Comment by samuelh2005** [comment]
There's https://github.com/North-Western-Development/oc2r, a 1.20.1 development of OC2.
